src/sentry/static/sentry/app/components/forms/selectControl.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
-import ReactSelect, {components as selectComponents} from 'react-select';
+import ReactSelect, {components as selectComponents, StylesConfig} from 'react-select';
 import Async from 'react-select/async';
 import AsyncCreatable from 'react-select/async-creatable';
 import Creatable from 'react-select/creatable';
 import {withTheme} from 'emotion-theming';
 
 import {IconChevron, IconClose} from 'app/icons';
 import space from 'app/styles/space';
+import {Choices} from 'app/types';
 import convertFromSelect2Choices from 'app/utils/convertFromSelect2Choices';
+import {Theme} from 'app/utils/theme';
 
 import SelectControlLegacy from './selectControlLegacy';
 
-const ClearIndicator = props => (
+const ClearIndicator = (props: any) => (
```

Comment:
Can you just use `React.ComponentProps<typeof selectComponents.ClearIndicator>`

Reply:
Maybe even better would be to declare the variable as 

`React.ComponentType<typeof selectComponents.ClearIndicator>`

Reply:
Maybe those types are poor too though 😬

Reply:
Yes. Currently that is a long way to say `any`. :cry: